src/cycax/cycad/engines/cycax_part_freecad.py

```diff
@@ -274,10 +276,24 @@ def render_to_dxf(self, active_doc: App.Document, view: str | None = None):
 
         importDXF.export(__objs__, str(f"{self.filepath}-{view}.dxf"))
 
+    def render_to_svg(self, active_doc: App.Document, view: str | None = None):
+        """This method will be used for creating a svg of the object currently in view.
+        Args:
+            active_doc: The FreeCAD document.
+            view: The side from which to produce the output file.
+        """
+        view_doc = FreeCADGui.activeDocument()
+        view = self.change_view(active_doc=view_doc, side=view, default="TOP")
+        FreeCADGui.SendMsgToActiveView("ViewFit")
+        __objs__ = []
+        __objs__.append(active_doc.getObject("Shape"))
+
+        importSVG.export(__objs__, str(f"{self.filepath}-{view}.svg"))
```

Comment:
Just check I though there was a built in. safeVector method in FreeCAD for saving svg.

Reply:
https://wiki.freecad.org/Draft_SVG
https://wiki.freecad.org/SVG : FreeCAD can read and export SVG graphics using the [Draft SVG](https://wiki.freecad.org/Draft_SVG) function from the Workbench [Draft](https://wiki.freecad.org/Draft_Workbench).svg Draft Workbench.

Reply:
I cannot say this is wrong, it just looks different than how we deal with PNG files. This is more like DXF where one has to jump through hoops. Also check that the files do have valid content.

Reply:
As far as I can tell the content is valid but the view is strange. I have just followed the suggestion in Draft.
![testing_side-right](https://github.com/tsolo-io/cycax/assets/134229763/78f0d8d7-9ab8-4165-b1e5-f45443fcd724)
![testing_side-front](https://github.com/tsolo-io/cycax/assets/134229763/06df517c-ada6-419a-9aee-654b2959d100)
![testing_side-bottom](https://github.com/tsolo-io/cycax/assets/134229763/d713dbe2-013f-4c99-bb6e-8a7718748453)
![testing_side-back](https://github.com/tsolo-io/cycax/assets/134229763/bd22b591-58f9-47ee-b1b6-8f1e216fe8f2)
![testing_side-top](https://github.com/tsolo-io/cycax/assets/134229763/344fdd23-a61c-4844-96b0-87fae111a27b)
![testing_side-left](https://github.com/tsolo-io/cycax/assets/134229763/620e2dbc-770c-4f40-b3f6-a91986c628b5)

Reply:
that is sick. It is the Draft view that concerns me since we are not in Draft workbench but in Part workbench. I guess that still validates that the sides are the same, so OK for now.